pdns/nsecrecords.cc

```diff
@@ -29,7 +29,9 @@ class NSECBitmapGenerator
 public:
   NSECBitmapGenerator(DNSPacketWriter& pw_): pw(pw_)
   {
+    memset(res, 0, 34);
```

Comment:
I prefer `sizeof(res)`
There's another occurrence of this memset.

Reply:
Done.